packages/rum-core/test/error-logging/error-logging.spec.js

```diff
@@ -220,7 +220,8 @@ describe('ErrorLogging', function() {
       colno
     })
     listener('error', {
-      message: 'Script error.' + testErrorMessage
+      message: 'Script error.' + testErrorMessage,
+      error: null
```

Comment:
Shouldn't we also test for undefined as well as null?

Reply:
Its always set to null during the creation https://www.w3.org/TR/html50/webappapis.html#dom-errorevent-error

Reply:
I vaguely remember that some browsers have/had a different behaviour even though the spec says it needs to be null!